android/src/main/java/com/reactnativemembrane/MembraneWebRTC.kt

```diff
@@ -609,23 +655,23 @@ class MembraneWebRTC(val sendEvent: (name: String, data: Map<String, Any?>) -> U
     }
 
     private fun emitAudioDeviceEvent(
-        audioDevices: List<AudioDevice>,
-        selectedDevice: AudioDevice?
+        audioDevices: List<AudioDevice>, selectedDevice: AudioDevice?
     ) {
-        val map = mapOf(
-            "selectedDevice" to (if (selectedDevice != null) audioDeviceAsRNMap(selectedDevice) else null),
-            "availableDevices" to audioDevices.map { audioDevice -> audioDeviceAsRNMap(audioDevice) }
-        )
+        val map =
+            mapOf("selectedDevice" to (if (selectedDevice != null) audioDeviceAsRNMap(selectedDevice) else null),
```

Comment:
The main key of this map is `selectedDevice` and the event id is `AudioDeviceUpdate`. Shouldn't they be the same?

Reply:
there are 2 main keys in this map